include/mega/megaclient.h

```diff
@@ -38,6 +38,136 @@
 
 namespace mega {
 
+class MEGA_API FetchNodesStats
+{
+public:
+    enum {
+        MODE_DB = 0,
```

Comment:
I agree, but I think that it's better to not change it in this case since it doesn't have any impact (there isn't any of those checks) and the specs of the stats have been already agreed with the server side, so I would prefer to not have to change them if there isn't a strong reason.